.github/workflows/ci.yml

```diff
@@ -19,8 +19,34 @@ jobs:
         env:
           GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
 
-  unit-test:
+  autoblack:
     needs: commit-lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Set up Python 3.7
+        uses: actions/setup-python@v1
+        with:
+          python-version: 3.7
+      - name: Install Black
+        run: pip install black
+      - name: Run black --check .
+        run: black --check .
+      - name: If needed, commit black changes to the pull request
+        if: failure()
+        run: |
+          black .
+          git config --global user.name 'autoblack'
+          git config --global user.email 'maximilianwerk@users.noreply.github.com'
```

Comment:
don't we have some `jina` account?

Reply:
I'll have a look. On the other hand, if we amend we might skip this completely and keep whoever committed beforehand. I'll have a look

Reply:
I would go for ammending but I would do it right before merge in CD.yml and not in CI.yml. (Appending commits in the middle of development process can be annoying for the developer?) Like they mayu continue working without pulling this commit and may get messy and conflicts they may not understand?